### DISCUSSION TITLE
"homematicip_local.set_cover_combined_position" don't work for shutter channels on blind actuator !?

### DISCUSSION THREAD

**Discussion by kitty99hub** [SukramJ/aiohomematic#1181] [Q&A] [answered]
Hi,
thanks for this great HA integration :)

I use a couple of Homematic blind actuators (HMip-DRBLI4).
It has 4 channels you can freely configure for shutters or blinds.
I use them for shutters and blinds in my house and will integrate them in Home Assistant via the custom homematic integration.
For controlling my blinds, in Home Assistant I create some button cards and use the command "homematicip_local.set_cover_combined_position" with the parameters "position" and "tilt_position".
This works very well.

But for my shutters it does not work!
Due to the fact that these channels are configured as shutter, they only have the functions "up" and "down" and no tilt function.
I tried to use the mentioned command with and without the tilt parameter, but it does not work.
Home Assistant will come out with following error message:

**Fehler beim Aufrufen des Dienstes homematicip_local/set_cover_combined_position. HaHomematicCover object has no attribute async_set_cover_combined_position**

Is there a service for the parameters of a shutter channel, or how can this be realized?

Many thanks
regards

**Comment by SukramJ** [comment, marked as answer]
[cover.set_cover_position](https://www.home-assistant.io/integrations/cover/#service-coverset_cover_position) ???

**Reply by kitty99hub** [reply]
Hi,
Thanks for your post ant this tip.
Shame on me that I didn't come across it myself.
I was probably to much focused on the HOMEMATIC services and was looking there for the right command. 
I had to play around a bit with the "cover.set_cover_position" command to integrate it into my code, but in the end it worked.
many thanks
